pyproject.toml

```diff
@@ -2,6 +2,68 @@
 requires = ["setuptools>=61.0"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "amulety"
+version = "0.1.1"
+description = "Python package to create embeddings of BCR amino acid sequences."
+readme = {file = "README.md", content-type = "text/markdown"}
+requires-python = ">=3.8, <4"
+license = {text = "MIT"}
+keywords = [
+    "immcantation",
+    "immunoinformatics",
+    "bioinformatics",
+    "embedding",
+    "antibody",
+    "BCR",
+    "Machine Learning",
+    "biology",
+    "NGS",
+    "next generation sequencing"
+]
+authors = [
+    {name = "Mamie Wang", email = "mamie.wang@yale.edu"},
+    {name = "Gisela Gabernet", email = "gisela.gabernet@yale.edu"},
+    {name = "Steven Kleinstein", email = "steven.kleinstein@yale.edu"}
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Topic :: Scientific/Engineering :: Bio-Informatics",
+]
+dependencies = [
```

Comment:
I like that you've moved the content of setup.py to the toml file! We have now here a duplicated list of dependencies though, inside the pyproject.toml and in the requirements.txt. Can the latter one be removed now? or is it still needed?